test/components/renku_data_services/message_queue/test_queue.py

```diff
@@ -42,21 +42,21 @@ async def fake_db_method(self, *, session, some_arg):
     fakerepo = FakeRepo()
     await fakerepo.fake_db_method(some_arg="test")
 
-    events = await app_config.redis.redis_connection.xrange("project.removed")
+    events = await app_config.redis.redis_connection.xrange("search.sync")
```

Comment:
suggestion(non-blocking): You can insert `QUEUE_NAME` from `components/renku_data_services/message_queue/converters.py` here if you wish rather than using a hardcoded value.

Reply:
Thank you!